src/tools/tidy/src/features.rs

```diff
@@ -423,6 +423,15 @@ fn map_lib_features(
                         continue;
                     }};
                 }
+
+                lazy_static::lazy_static! {
+                    static ref COMMENT_LINE: Regex = Regex::new(r"\s*//").unwrap();
```

Comment:
Shouldn't this have a starting anchor, so it avoids ignoring `let s = "//";`?
```suggestion
                    static ref COMMENT_LINE: Regex = Regex::new(r"^\s*//").unwrap();
```

Reply:
Also, there's a more accurate check in https://github.com/rust-lang/rust/blob/06f0adb34570ba83fee391abeb0bec0eec28a234/src/tools/tidy/src/style.rs#L81 - maybe these two checks should be combined?

Reply:
The other check is different enough (it looks for things *after* a comment) that it's probably better to leave them separate